index.html

```diff
@@ -1032,6 +1075,36 @@ <h2 class="code-line" data-line-start=9 data-line-end=10><a id="Companies_that_r
 					<td>Recruiter</td>
 					<td>02.04.22</td>
 				</tr>
+				<tr>
+					<td style="text-align:center"><a href="https://jobs.bytedance.com/en/position?keywords=&category=6704215862603155720&location=CT_163&project=&type=1&job_hot_flag=&current=1&limit=10">ByteDance</a></td>
```

Comment:
Thanks for adding this, would it be a good idea to merge ByteDance with Ticktok? We already got a referral for them.